content/blog/query-kubernetes/index.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Introducing CloudQuery
+authors: ["joe-duffy"]
+tags: ["Pulumi-News", "Kubernetes"]
+meta_desc: "Today we announced Pulumi CloudQuery, an SDK for programmatically querying cloud resoures."
+date: "2019-11-14"
+meta_image: "pulumi-crosswalk-k8s.png"
+---
+
+Today, as part of our Crosswalk for Kubernetes announcement[link], we're introducing CloudQuery for
+Kubernetes.
+
+CloudQuery is a relational TypeScript SDK for querying live Kubernetes resources. It makes it easy
+to answer questions that would normally be answered by hand-scripting calls to `kubectl`. Questions
+like:
+
+* How many distinct versions of MySQL are running in my cluster?
+* Which Pods are scheduled on nodes with high memory pressure?
+* Which Pods are publicly exposed to the internet via a load-balanced Service?
+
+CloudQuery supports both batch and streaming query modes. Batch queries run on a finite set of
```

Comment:
The idea that this builds on top of the Pulumi object model seems to be missing.  That seems important to tie this in to the rest of the Pulumi ecosystem, and in part to explain how this fits in with the `pulumi query` command.

Reply:
Hmm, I don't think it's that important for Kubernetes, because the Pulumi object model is the Kubernetes resource model? I also think the importance is mostly strategic, and important to us—getting people to use our toolchain.